ts/Core/Axis/Axis.ts

```diff
@@ -3934,10 +3934,20 @@ class Axis {
                 tickPositions.forEach(function (pos: number, i: number): void {
                     axis.renderTick(pos, i, slideInTicks);
                 });
+
                 // In a categorized axis, the tick marks are displayed
                 // between labels. So we need to add a tick mark and
                 // grid line at the left edge of the X axis.
-                if (tickmarkOffset && (axis.min === 0 || axis.single)) {
+                if (
+                    !axis.series.find( // #20166
+                        (series): string | number | false | undefined => (
+                            series.is('column') &&
```

Comment:
You should fix this for all cartesian series types. Instead of `series.is(...` check `series.xAxis`.

Reply:
Good catch! 

It now checks for `series.xAxis` and unit-test has been altered to check for this as well.

Might not have come out clearly in the issue but this is the only example which I was able to replicate both in jsfiddle as well as utils.

I have however uncovered a similar issue for the bar series, but it does seem to arise from the same conditions (`tick[-1]` being rendered for cartesian series using `pointPlacement`)

Should I create a seperate issue for the bar series bug?

Reply:
The bar series is cartesian, but it has inverted axis. This needs taken into account and is probably related.
See https://api.highcharts.com/highcharts/chart.inverted